src/instructlab/model/evaluate.py

```diff
@@ -516,6 +516,12 @@ def launch_server(
     is_flag=True,
     help="Print serving engine logs.",
 )
+@click.option(
+    "-ss",
+    "--skip-server",
+    is_flag=True,
+    help="Skip launching the server (without using backend e.g. vLLM or llama-cpp) and evaluate directly with the HuggingFace model. This option supports mmlu and mmlu_branch benchmarks.",
```

Comment:
MMLU isn't supported by `llama-cpp` since we can't read the logprobs, so we can get rid of this part

```suggestion
    help="Skip launching the server and evaluate directly with the HuggingFace model. This option supports mmlu and mmlu_branch benchmarks.",
```

Reply:
updated